src/Sonarr.Api.V3/Config/ImportListConfigController.cs

```diff
@@ -0,0 +1,26 @@
+using FluentValidation;
+using NzbDrone.Core.Configuration;
+using NzbDrone.Core.Validation;
+using Sonarr.Http;
+
+namespace Sonarr.Api.V3.Config
+{
+    [V3ApiController("config/importlist")]
+
+    public class ImportListConfigController : ConfigController<ImportListConfigResource>
+    {
+        public ImportListConfigController(IConfigService configService)
+            : base(configService)
+        {
+            SharedValidator.RuleFor(x => x.ListSyncTag)
+               .ValidId()
+               .WithMessage("Tag must be specified")
+               .When(x => x.ListSyncLevel == NzbDrone.Core.ImportLists.ListSyncLevelType.KeepAndTag);
```

Comment:
Add a using statement for `NzbDrone.Core.ImportLists` and change to `ListSyncLevelType.KeepAndTag`